vignettes/MSstatsUpdated.Rmd

```diff
@@ -0,0 +1,324 @@
+---
+title: "MSStats Vignette"
+date: September 5th, 2024
+---
+
+```{r style, echo = FALSE, results = 'asis'}
+BiocStyle::markdown()
+```
+
+```{r global_options, include=FALSE}
+knitr::opts_chunk$set(fig.width=10, fig.height=7, warning=FALSE, message=FALSE)
+options(width=110)
+```
+
+```{=html}
+<!--
+%\VignetteIndexEntry{MSstats: Protein/Peptide significance analysis}
```

Comment:
Looking at this [R documentation guidelines](https://r-pkgs.org/vignettes.html), vignetteIndexEntry should be the same as the title.  For now, I think you can replace this with `MSstats Vignette`

Reply:
Or maybe we can change the title + vignetteIndexEntry to a more meaningful title, like MSstats End to End Workflow